core/libs/commonwealth/commonwealth/utils/DHCPServerManager.py

```diff
@@ -60,6 +63,8 @@ def command_list(self) -> List[Union[str, pathlib.Path]]:
             self.binary(),
             "--no-daemon",
             f"--interface={self._interface}",
+            f"--dhcp-range={self._ipv4_network_prefix()}.100,{self._ipv4_network_prefix()}.200,255.255.255.0,24h",
```

Comment:
It would be nice if such magical numbers could exist inside a variable with a good name and a comment to explain what such numbers are for.

Reply:
Agree. Done.